# HG changeset patch
# User Rik <rik@octave.org>
# Date 1545608958 28800
#      Sun Dec 23 15:49:18 2018 -0800
# Branch stable
# Node ID 24d8411954c8a7d55f7725d7923f99d4485c3e16
# Parent  3c5863948bb34a97088bc7154723a581cc21017a
doc: Add new words to Octave-specific spelling dictionary.

* aspell-octave.en.pws: Add new words to Octave-specific spelling dictionary.

diff --git a/doc/interpreter/doccheck/aspell-octave.en.pws b/doc/interpreter/doccheck/aspell-octave.en.pws
--- a/doc/interpreter/doccheck/aspell-octave.en.pws
+++ b/doc/interpreter/doccheck/aspell-octave.en.pws
@@ -4,16 +4,17 @@ accumarray
 acknowledgements
 Acknowledgements
 adams
 Affero
 afterwards
 al
 aleph
 amd
+amongst
 anisotropic
 API
 APIs
 appdata
 approximant
 arg
 args
 ARMA
@@ -28,36 +29,36 @@ atan
 Attr
 audiodevinfo
 audioplayer
 audiorecorder
 AutoCAD
 Autoconf
 autocorrelated
 autocovariances
+Autoload
 autoload
-Autoload
 autoloaded
 Autoloading
 Automake
 autoregression
 autoregressions
 autoscaled
 AutoScaling
 AWK
 azimuthal
 backend
 backgroundcolor
 backtrace
 bartlett
 Barycentric
 basevalue
 BaseValue
+BDF
 bdf
-BDF
 benchmarking
 betacdf
 betainc
 betainv
 betaln
 betapdf
 betarnd
 BFGS
@@ -79,18 +80,18 @@ boxoff
 boxon
 bracketx
 brackety
 braindead
 breakpoint
 broadcastable
 BufSize
 builtin
+Bulleted
 bulleted
-Bulleted
 ButtonDownFcn
 buttondownfcn
 BV
 BZ
 bzip
 CallbackObject
 callee's
 camd
@@ -141,18 +142,18 @@ colamd
 CollectOutput
 colorbar
 colorcube
 colormap
 colormaps
 ColorOrder
 colororder
 colperm
+commentstyle
 CommentStyle
-commentstyle
 ComplexEqn
 cond
 condest
 conformant
 cong
 const
 contextless
 contourc
@@ -163,25 +164,25 @@ CorelDraw
 corrcoef
 cosecant
 courseware
 cov
 covariance
 createcancelbtn
 CreateFcn
 CRLF
+Crout
 crout
-Crout
 cryptographic
 CSparse
 CSV
 csymamd
 ctranspose
+Ctrl
 CTRL
-Ctrl
 cubehelix
 cummax
 cummin
 cumprod
 cumsum
 cURL
 CurrentAxes
 CurrentFigure
@@ -218,18 +219,18 @@ DeleteFcn
 deletefcn
 delim
 deltaX
 demi
 det
 dggsvd
 diag
 diagcomp
+dialogs
 Dialogs
-dialogs
 diamondsuit
 differentiable
 digamma
 dimensionality
 dimensionally
 dir
 disp
 displayname
@@ -263,35 +264,35 @@ eigenvector
 eigenvectors
 eigs
 elementwise
 elseif
 emacs
 emptyvalue
 EmptyValue
 endfunction
+Endian
 endian
-Endian
 endif
+endofline
 EndOfLine
-endofline
 EOF
 EOLs
 eps
 eq
 equidistributed
+Equilibration
 equilibration
-Equilibration
 equispaced
 erf
 erfc
 erfi
 errno
+Errorbar
 errorbar
-Errorbar
 errorbars
 Errorbars
 errordlg
 ErrorHandler
 ESC
 et
 etree
 etreeplot
@@ -301,28 +302,28 @@ Executables
 Exif
 exitflag
 expcdf
 expinv
 exppdf
 exprnd
 extendedtext
 extrema
+FaceColor
 facecolor
-FaceColor
 FaceLighting
 FaceNormals
+facevertexcdata
 FaceVertexCData
-facevertexcdata
 factorizations
 fcdf
 femmodel
 FFF
+fft
 FFT
-fft
 fftn
 fftpack
 FFTs
 fftw
 fieldname
 fieldnames
 FIFOs
 filename
@@ -347,23 +348,24 @@ forall
 foregroundcolor
 formfeed
 Fortran
 fpdf
 fprintf
 FreeBSD
 FreeSans
 freespacing
+freetype
 FreeType
-freetype
 frnd
 Fs
 FSF
 fullpath
 fullpathext
+func
 FunValCheck
 gamcdf
 gaminv
 gampdf
 gamrnd
 gaussian
 gca
 gcbo
@@ -383,22 +385,22 @@ gesvd
 gfortran
 Ghostscript
 Ghostscript's
 gif
 GIF
 glibc
 globbing
 glpk
+GLS
 gls
-GLS
 glyphs
 GMRES
+Gnuplot
 gnuplot
-Gnuplot
 Goto
 goto
 gouraud
 gperf
 GPL
 GPLK
 gplot
 grabdemo
@@ -453,18 +455,18 @@ hygecdf
 hygeinv
 hygepdf
 hygernd
 hypervolume
 ichol
 ict
 IDE
 IEC
+ieee
 IEEE
-ieee
 ifelse
 ifft
 ifftn
 ignorecase
 ij
 ilu
 ilutp
 Im
@@ -523,21 +525,21 @@ Jacobian
 Jacobians
 javaaddpath
 javamem
 Javascript
 jbig
 JConstant
 JDK
 JIS
+JIT
 jit
-JIT
 JPattern
+JPEG
 jpeg
-JPEG
 jpg
 jvm
 JVM's
 kendall
 keybindings
 keypress
 keypressfcn
 KeyType
@@ -585,18 +587,18 @@ LLVM
 LM
 loadpath
 logarithmically
 logfile
 logncdf
 logninv
 lognpdf
 lognrnd
+Lookup
 lookup
-Lookup
 lookups
 lossless
 lsode
 lt
 lu
 luminance
 lvalue
 lvl
@@ -647,47 +649,47 @@ Minimizers
 Minitab
 minval
 MIP
 miscompiled
 mkdir
 mkoctfile
 mldivide
 mmd
+MMM
 mmm
-MMM
 mmmm
 mmmyy
 mmmyyyy
 movegui
 mpoles
 mpower
 mput
 mrdivide
 msgbox
 MStateDependence
 MSYS
 mtimes
+Multi
 multi
-Multi
 multibyte
 multipage
+multipledelimsasone
 MultipleDelimsAsOne
-multipledelimsasone
 MultiSelect
 multistep
 MvPattern
 mxArray
 myclass
 myfun
 nabla
 NAMESPACE
 namespace
+NaN
 nan
-NaN
 nanflag
 NaNs
 nargin
 nargout
 nbincdf
 nbininv
 nbinpdf
 nbinrnd
@@ -749,46 +751,46 @@ ofig
 Ogg
 ogg
 OKString
 OLS
 omitnan
 onCleanup
 online
 OpenBLAS
+opengl
 OpenGL
-opengl
 OpenJDK
 oplus
 optimizations
 Ord
 oregonator
 Orthogonalize
 orthonormal
 oslash
 otimes
 outerposition
 outlier
 outliers
+OutputFcn
 outputfcn
-OutputFcn
 OutputSel
 overdetermined
 overridable
 paperorientation
 paperposition
 PaperPosition
 paperpositionmode
 papersize
+PaperUnits
 paperunits
-PaperUnits
 parametrically
 parseparams
+PBM
 pbm
-PBM
 PBMplus
 pc
 pcg
 PCG
 pchip
 pcr
 PCRE
 pcre
@@ -798,18 +800,18 @@ PDF
 pdflatex
 pentadiagonal
 periodogram
 perp
 PGF
 PGMRES
 PHP
 pict
+Piecewise
 piecewise
-Piecewise
 pinv
 PixelRegion
 PNG
 png
 poisscdf
 poissinv
 poisspdf
 poissrnd
@@ -825,42 +827,43 @@ POSDEF
 POSIX
 postorder
 PostScript
 powerset
 pre
 preallocate
 preconditioner
 preconditioners
+preformatted
 Preformatted
-preformatted
 premultiplied
 Prepend
 prepended
 preselected
 presolver
 printf
 priori
+Profiler
 profiler
-Profiler
 programmatically
 prolate
 PromptString
 propto
 ps
 pseudoinverse
 pseudonorm
 pseudorandom
 Pseudospectra
 pseudospectrum
 ptrace
 pushbutton
+pushbuttons
 Pxx
+Qhull
 qhull
-Qhull
 QP
 QQ
 QRUPDATE
 qrupdate
 QScintilla
 quadcc
 quadgk
 quadl
@@ -869,48 +872,49 @@ quadv
 Quantile
 quantile
 quantiles
 quantization
 quartic
 quartile
 questdlg
 Quickhull
+QZ
 qz
-QZ
 radian
 radians
 radices
 radiobutton
 radix
 rande
 randg
 randi
 randn
 randp
 randperm
 rangle
 ranlib
 rceil
 rdivide
+readline
 Readline
-readline
 RECT
 recursing
 redistributable
 reentrant
 reflectance
 regex
 regressor
 relicensing
 RelTol
 renderer
 renderers
 repelems
 replacechildren
+ReplacementStyle
 repmat
 representable
 reproducibility
 resampled
 resampling
 resize
 Resize
 resized
@@ -1000,16 +1004,17 @@ sT
 stairstep
 Startup
 startup
 statinfo
 stdin
 stdout
 stepsize
 STFT
+str
 strcmp
 stringanchors
 strncmp
 strncmpi
 strread
 strread's
 strsplit
 struct
@@ -1017,35 +1022,35 @@ structs
 subarrays
 subclasses
 Subclasses
 subdiagonal
 subdiagonals
 subdirectories
 subdirectory
 subexpressions
+subfunction
 Subfunction
-subfunction
 Subfunctions
 subfunctions
 subinterval
 subintervals
 Subintervals
 sublicenses
 Sublicensing
 submatrices
 submatrix
 submenu
 suboptimal
 subprocess
+Subprocesses
 subprocesses
-Subprocesses
 subsasgn
+Subscripted
 subscripted
-Subscripted
 subscripting
 subseteq
 subsindex
 subspaces
 Subspaces
 subsref
 substring
 substrings
@@ -1061,22 +1066,22 @@ superscripted
 superscripting
 supset
 supseteq
 surd
 SV
 svd
 SVD
 svds
+svg
 SVG
-svg
 sym
 Sym
+symamd
 SYMAMD
-symamd
 symbfact
 symrcm
 Syntaxes
 tcdf
 terminal's
 tex
 texi
 texinfo
@@ -1103,16 +1108,17 @@ tinv
 tmp
 Toeplitz
 togglebutton
 togglebuttons
 tokenExtents
 TolF
 TolFun
 TolX
+toolchain
 Toolkits
 toolkits
 toplevel
 TP
 tpdf
 traceback
 TRANSA
 trapz
@@ -1128,49 +1134,52 @@ trisurf
 trivariate
 trnd
 TrueColor
 truecolor
 tuples
 txi
 typedefs
 TypicalX
+UB
 ub
-UB
 uchar
 UHESS
 UI
+uibuttongroup
 Uibuttongroup
-uibuttongroup
 uibuttongroups
+Uicontextmenu
 uicontextmenu
-Uicontextmenu
+uicontrol
 Uicontrol
-uicontrol
 uicontrols
 UID
 uimenu
 uint
 uipanel
 Uipanel
 uipanels
 uipushtool
 Uipushtool
 uipushtools
 uiputfile
+uitab
+uitable
+Uitable
+uitoggletool
 Uitoggletool
-uitoggletool
 uitoolbar
 Uitoolbar
 ulong
 Ultrix
 umfpack
 uminus
+Unary
 unary
-Unary
 unconvertible
 undirected
 unifcdf
 unifinv
 UniformOutput
 UniformValues
 unifpdf
 unifrnd
@@ -1218,25 +1227,25 @@ versa
 vertcat
 VertexNormals
 verticalalignment
 vertices
 viewable
 viridis
 vorbis
 Vorbis
+Voronoi
 voronoi
-Voronoi
 waitbar
 waitbars
 warndlg
 WAV
 wav
+WayPoints
 Waypoints
-WayPoints
 waypoints
 wblcdf
 wblinv
 wblpdf
 wblrnd
 WestOutside
 whitespace
 Whitespace
@@ -1244,18 +1253,18 @@ whos
 wienrnd
 Wikipedia
 wildcard
 Wildcards
 wildcards
 windowbuttondownfcn
 windowbuttonmotionfcn
 windowbuttonupfcn
+WindowStyle
 windowstyle
-WindowStyle
 WIPO
 wireframe
 workspace
 wp
 xb
 xboxes
 xc
 xcorr
